symengine/functions.h

```diff
@@ -924,6 +924,30 @@ class Erf : public OneArgFunction
 //! Canonicalize Erf:
 RCP<const Basic> erf(const RCP<const Basic> &arg);
 
+class Erfc : public OneArgFunction
+{
+    /*   The complementary error function. This function is defined as:
+     *
+     *   .. math::
+     *      \mathrm{erfc}(x) =1 - \frac{2}{\sqrt{\pi}} \int_0^x e^{-t^2}
+     *\mathrm{d}t.
```

Comment:
I would put a space after `=`, and also indent the second line with `\mathrm{d}t.` to the right, something like this:
```
*                \mathrm{d}t.
```